pyrefinebio/token.py

```diff
@@ -48,9 +48,7 @@ def agree_to_terms_and_conditions(cls, api_token):
             api_token (str): the uuid string identifying the token
                              you want to activate.
         """
-
-        response = put_by_endpoint("token/" + api_token, payload={"is_activated": True})
-        return response
+        put_by_endpoint("token/" + api_token, payload={"is_activated": True})
```

Comment:
```suggestion
        return put_by_endpoint("token/" + api_token, payload={"is_activated": True})
```
I think returning the response here could be helpful in understanding what went wrong.